tests/Core/Models/ByteTest.php

```diff
@@ -44,4 +44,8 @@ protected function tearDown(): void {
         parent::tearDown();
         $this->byteObject = null;
     }
+
+    public function testCanGetByteValue(): void {
+        $this->assertEquals(200, $this->byteObject->getProperties()['size']->getValue());
```

Comment:
Suggestion for the unit tests...
It would be cleaner if unit tests for Byte only tested Byte's functionality e.g. serialize/deserialize a Byte object directly instead of going via an Entity or assert `(new Byte(200)->getValue())` equals 200 etc. This way we only test one unit and not its integration with another component
(Applies for the other unit test classes)